spec/models/miq_provision_request_spec.rb

```diff
@@ -2,19 +2,31 @@
 
 describe MiqProvisionRequest do
   it ".request_task_class_from" do
```

Comment:
¯\\\_(ツ)_/¯ ... good chance they don't actually all need to stay, once the method is back to a single implementation. I just needed to expand it for now, to drive the several different branches it can currently take to find the right thing.